.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,61 +8,52 @@ title: Manta {{ SET_VERSION }} Release checklist
-- [ ] Verify [benchmarks](#benchmarks) have been updated for any modified
-    runtime logic.
-- [ ] Check for any upstream storage migrations and perform tests with `try-runtime`, if any.
-- [ ] Update hard-coded URLs to polkadot/manta binaries/runtimes in `publish_draft_releases.yml` CI workflow.
-
-The following checks can be performed after we have frozen our release candidate:
-
-- [ ] Code freeze should typically happen one week prior to release, to ensure we have enough time for related testing.
-- [ ] Notify everyone, especially people with merge rights to `manta` (stechu, Dengjianping) that a release is ongoing and no more merges to `manta` should happen until told otherwise
-- [ ] Complete the [manual QA workflow](https://www.notion.so/mantanetwork/d55be01354bb4f579b16d6e34df9e2e1?v=dcfa54e2b4a343ad9b899574ddb94a1c).
+- [ ] Announce code freeze (typically one week prior to release), to ensure we have enough time for related testing. Notify everyone @Runtime that a release is ongoing and no more merges to `manta` should happen until told otherwise
+- On a branch named `release-vX.Y.Z` or `release-vX.Y.Z-something` ( something could be e.g. `alpha` or `rc1` ). Substitute X Y Z with the release version number.
+    - [ ] Verify that each crate's `version` has been bumped from previous release.
+    - [ ] Verify [`spec_version`](#spec-version) has been incremented since the
+        last release for any native runtimes from any existing use on public
+        (non-private/test) networks. If the runtime was published (release or pre-release), either
+        the `spec_version` or `impl` must be bumped.
+    - [ ] Verify pallet and [extrinsic ordering](#extrinsic-ordering) has stayed
+        the same. Bump `transaction_version` if not.
+    - [ ] Verify new extrinsics have been correctly whitelisted/blacklisted
+    - [ ] Verify [benchmarks](#benchmarks) have been updated for any modified
+        runtime logic.
+    - [ ] Check for any upstream storage migrations and perform tests with `try-runtime`, if any.
+    - [ ] Update hard-coded URLs to `polkadot` AND `manta` binaries/runtimes in `publish_draft_releases.yml` CI workflow.
+    - [ ] Generate new changelog using `dev-tools` repo
+    - [ ] Merge when green and reviewed
+    - [ ] Tag the release
+    - [ ] Check that a draft release has been created at https://github.com/Manta-Network/Manta/releases with relevant [release notes](#release-notes)
+    - [ ] Check that build artifacts have been added to the
+        draft-release
+    - [ ] Create a Pre-Release on github and add Release Notes (see below for what needs to be in them)
+
+# Deploy to internal testnets ( fast runtime )
 - [ ] Verify Polkadot JS API are up to date with the latest
     runtime changes.
-- [ ] Execute runtime upgrade to Baikal relay and verify network stability.
+- [ ] Execute client upgrade on Baikal relaychain nodes if needed
+- [ ] Execute runtime upgrade to Baikal relaychain and verify network stability.
```

Comment:
Rename Baikal to kusama-internal or kusama-staging maybe

Reply:
Kusama internal is the public testnet.

I'm counting on us relaunching a fastamari-on-baikal as internal testnet soon

Reply:
Or are you planning on fully retiring the Baikal relay and running everything on kusama-internal (which also needs to be renamed now)

Reply:
I mean whatever new pre-staging we deploy (fast or not) it would also be on the same relay chain, which is not called Baikal anymore, but Kusama-internal.

Reply:
Are you sure? This would mean we have to have a customized runtime with changed paraID just for this deployment.

Not sure saving 3 compute nodes to run 15 validators on is worth that, deploying the network is fast with ansible and we can reset the pre-staging relay whenever sync time gets annoying